evap/rewards/tests/test_views.py

```diff
@@ -79,13 +79,40 @@ def test_redeem_points_for_expired_event(self):
         self.assertContains(response, "event expired already.")
         self.assertEqual(5, reward_points_of_user(self.student))
 
+    def helper_post_redemption_request(self, redemption_params, additional_params=None, status=400):
```

Comment:
Now, we never use the `status` parameter; should we remove it and instead call the function `post_invalid_redemption_request` or so, to encode that we expect an error?

Reply:
(or better, add some cases where we use this function and the request is successful; then, the default of the helper probably shouldn't be 400, but something positive)